### PR TITLE
docs: description client-control plugin configuration `max_body_size` unit

### PR DESCRIPTION
**Pull request by luoluoyuyu**
… unit

### Description

description client-control plugin configuration `max_body_size` unit

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/plugins/client-control.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ This Plugin requires APISIX to run on APISIX-Runtime. See [apisix-build-tools](h
 
 | Name          | Type    | Required | Valid values | Description                                                                                                                          |
 | ------------- | ------- | -------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
-| max_body_size | integer | False    | [0,...]      | Dynamically set the [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) directive. |
+| max_body_size | integer | False    | [0,...]      | Set the maximum limit for the client request body and dynamically adjust the size of [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size), measured in bytes. |
```
Comment (moonming) [inline docs/en/latest/plugins/client-control.md:43]: - The unit of the [variable](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) is `m`, why `max_body_size` is `bytes` here?
- What does it mean to set `max_body_size` to 0?
- The corresponding code also needs to be commented.

**Review by Vacant2333 — APPROVED** [review]

File: docs/en/latest/plugins/client-control.md (hunk 1 above)
Comment (luoluoyuyu) [reply docs/en/latest/plugins/client-control.md:43]: Hi @moonming 
For a description of the units check this document https://nginx.org/en/docs/syntax.html, in https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_ The unit of `client_max_body_size` is not specified in size, but the default value is `1m`. If you configure `client_max_body_size 1024` in the configuration file, then the maximum request body size is 1024 bytes.

Comment (luoluoyuyu) [reply docs/en/latest/plugins/client-control.md:43]: > What does it mean to set max_body_size to 0?
> The corresponding code also needs to be commented.

That's a great suggestion.

Comment (kayx23) [reply docs/en/latest/plugins/client-control.md]: [`Content-Length`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Length) is in bytes and is used in comparison with the configured `max_body_size` here:

https://github.com/apache/apisix/blob/4db435ee38b425da6364ad41aa9c019046190d87/apisix/plugins/client-control.lua#L57

File: apisix/plugins/client-control.lua [hunk 2]
```diff
@@ -26,6 +26,7 @@ local schema = {
         max_body_size = {
             type = "integer",
             minimum = 0,
+            description = "Maximum header length in bytes, no check if the value is 0"
```
Comment (kayx23) [inline apisix/plugins/client-control.lua:29]:
```suggestion
            description = "Maximum message body size in bytes. No restriction when set to 0."
```

**Review by kayx23 — APPROVED** [review]

**Review by monkeyDluffy6017 — DISMISSED** [review]

File: docs/en/latest/plugins/client-control.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ This Plugin requires APISIX to run on APISIX-Runtime. See [apisix-build-tools](h
 
 | Name          | Type    | Required | Valid values | Description                                                                                                                          |
 | ------------- | ------- | -------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
-| max_body_size | integer | False    | [0,...]      | Dynamically set the [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) directive. |
+| max_body_size | integer | False    | [0,...]      | Set the maximum limit for the client request body and dynamically adjust the size of [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size), measured in bytes. When `client_max_body_size` is set to 0, no check will be performed on the client request body size. |
```
Comment (moonming) [inline docs/en/latest/plugins/client-control.md:43]:
```suggestion
| max_body_size | integer | False    | [0,...]      | Set the maximum limit for the client request body and dynamically adjust the size of [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size), measured in bytes. If you set the `max_body_size` to 0, then the size of the client's request body will not be checked. |
```

File: docs/zh/latest/plugins/client-control.md [hunk 4]
```diff
@@ -38,9 +38,9 @@ description: 本文介绍了 Apache APISIX proxy-control 插件的相关操作
...
-| max_body_size | integer        | 否    | [0,...] | 动态设置 [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) 的大小。 |
+| 名称      | 类型          | 必选项 | 有效值                                                                    | 描述                                                                                                                                                                                    |
+| --------- | ------------- | ----------- | ------------------------------------------------------------------------ |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| max_body_size | integer        | 否    | [0,...] | 设置客户端请求体的最大上限，动态调整 [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) 的大小，单位为字节。当 `client_max_body_size` 为 0 时，将不会对客户端请求体大小进行检查。 |
```
Comment (moonming) [inline docs/zh/latest/plugins/client-control.md:43]:
```suggestion
| max_body_size | integer        | 否    | [0,...] | 设置客户端请求体的最大上限，动态调整 [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) 的大小，单位为字节。当设置`max_body_size` 为 0 时，将不会对客户端请求体大小进行检查。 |
```

**Review by moonming — CHANGES_REQUESTED** [review]

**Review by Sn0rt — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Review by kayx23 — APPROVED** [review]

**Pull request merged**